_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,62 @@
 title: Banner Component
 ---
 
-The purpose of the Banner component is to render a media in a given page slot. The media can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
+The banner component is used to render a media item in a given page slot. The media item can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
 
 ## CMS Component Binding
 
-There are multipe banner component types in the CMS:
+There are multiple banner component types in the CMS, as follows:
 
--   `BannerComponent`
--   `SimpleBannerComponent`
--   `SimpleResponsiveBannerComponent`
+- `BannerComponent`
+- `SimpleBannerComponent`
+- `SimpleResponsiveBannerComponent`
 
-There is also a `RotatingImagesComponent` which renders a one or multiple `BannerComponents` in a carousel.
+There is also a `RotatingImagesComponent`, which renders one or multiple instances of `BannerComponent` in a carousel.
```

Comment:
> There is also a `RotatingImagesComponent`, which renders one or multiple instances of `BannerComponent` in a carousel.

Can we rephrase: 

There is also a `RotatingImagesComponent`, which is a so-called "container component". A container component holds multiple components, the `RotatingImagesComponent` is used to render multiple `BannerComponent` in a carousel.

Reply:
👍